player/tests/data/quad.ron

```diff
@@ -86,7 +86,7 @@
             ],
             depth_stencil_state: None,
             vertex_state: (
-                index_format: Uint16,
+                index_format: Some(Uint16),
```

Comment:
this should actually be `None`. We only need to specify this for strip topologies